pop-api/README.md

```diff
@@ -1,73 +1,64 @@
+blockchain built to power innovative and impactful solutions on Polkadot. Designed for stability, simplicity and
+efficiency, the API abstracts away the complexities of the runtime, enabling developers to focus on building powerful
+applications rather than managing intricate blockchain details.
 
-## What is the Pop API?
+### Design Goals
 
-One of the core value propositions of Pop Network is to enable smart contracts to easily access the power of Polkadot. As such, the Pop API was built to enable smart contracts to easily utilize the functionality provided by the Pop Network parachain runtime.
+- **Simple**: enhance the developer experience by abstracting away the complexities of runtime functionality, making it
+  easy for developers to build advanced applications.
+- **Versioned**: offer a stable, versioned interface that ensures smart contracts stay compatible as the runtime
+  evolves, enabling seamless integration of new features without disrupting existing contracts.
+- **Efficient**: optimise for minimal contract size, having the lowest possible deployment and execution costs.
 
-Substrate already exposes a Runtime API which is typically associated with the “outer node” calling into the runtime via RPC. Pop Network extends this concept in a direction that makes it usable by smart contracts, allowing untrusted contracts to make use of carefully exposed trusted functionality available within the runtime - this is the Pop API.
+### Key Features
 
-The Pop API is designed to be:
-- **Stable**: contracts are generally immutable and therefore need a stable interface, especially as the runtime evolves over time.
-- **Future Proof**: technologies improve, so the Pop API implementation is adaptable to new approaches without affecting the interface. The Pop API implementation also selects the optimal approach to realize a use case from a range of possible technical implementations.
-- **Versioned**: not every future technology can be retro-fitted, so the API is versioned.
-- **Simple**: providing an abstraction layer over runtime-level features. Hand-rolled with love, Pop API is easy to use for smart contract development.
+- **Versioned Interface**: Provides backward compatibility, ensuring that existing contract functionality remains stable
+  as new features are added to the runtime.
+- **Error Handling**: Supports rich, versioned error types, enabling contracts to receive and interpret any runtime
+  error, making troubleshooting and development easier.
+- **Use Cases**:
+    - [Fungibles](./src/v0/fungibles/README.md): Interacting and managing fungible tokens.
+    - In development:
+        - Non Fungibles: Interacting and managing non fungible tokens.
+        - Messaging: Cross chain rails for interaction with other chains using ISMP & XCM.
+        - Sponsorship: Allowing smart contracts to sponsor transactions.
+        - Incentives: Incentivise smart contracts by sharing chain revenue.
```

Comment:
maybe is incentive the right word? Incentives are what it is internally, but what is the builder getting?

My recommendation: "Revenue Sharing: Pop Network shares transaction fees with smart contracts"

Reply:
Yeah maybe the naming needs a tweak. I think this is an inherent problem of including topics in the documentation that are under development.

I would suggest leaving this as is for now and address with the inclusion of the pallet to master ?

Reply:
The reason I added those was to show that we are actively developing features. In addition, just adding Fungibles and Nonfungibles also doesn't really show that we are innovating. 

I agree with Ale that changing the name doesn't matter that much now, the description is clear of what the use case is about.